python/tvm/micro/model_library_format.py

```diff
@@ -494,8 +500,10 @@ def _export_graph_model_library_format(
                 devices,
                 workspace_size,
                 include_path,
-                inputs_sizes,
-                output_sizes,
+                # inputs_sizes,
+                # output_sizes,
```

Comment:
```suggestion
```

(Just delete these, no need to leave them here)

Reply:
ACK